packages/workflow/src/internals.ts

```diff
@@ -377,7 +391,7 @@ export class State {
   /**
    * Source map file for looking up the source files in response to __enhanced_stack_trace
    */
-  public sourceMap: unknown;
+  public sourceMap: any;
```

Comment:
Wondering if there's a type definition for this in one of our dependencies that we can use and avoid the `any` type.

Reply:
Maybe this: 
`public sourceMap: SourceMap | undefined;`